module/Finna/src/Finna/Search/Factory/PrimoBackendFactory.php

```diff
@@ -82,6 +82,15 @@ protected function createConnector()
             $this->serviceLocator->get(\VuFind\Cache\Manager::class)
         );
 
+        if ($extraParams = ($this->primoConfig->General->extraHttpParams ?? [])) {
```

Comment:
Tämän if:n voisi hyvin jättää pois ja laittaa suoraan `foreach ($this->primoConfig->General->extraHttpParams ?? [] as $param)`.